components/ModEditSubmissionForm.vue

```diff
@@ -657,6 +684,197 @@ const listPrefectureJapanJa: Ref<string[]> = ref([
 const formSubmissionId = moderationSubmissionStore.selectedSubmissionId
 moderationSubmissionStore.filterSelectedSubmission(formSubmissionId)
 
+// Tracks whether we want to display adding a new name
+const addingLocaleName: Ref<boolean> = ref(false)
+// Tracks how we want to display editing a Locale name
+const editingLocaleName: Ref<boolean> = ref(false)
+// Keeps track of which index of the name locale was clicked
+const chosenLocaleIndex: Ref<number | null> = ref(null)
+// Keeps track of the chosen healthcare professional to edit
+const chosenHealthcareProfessionalToEdit: Ref<LocalizedNameInput | undefined> = ref()
+// This keeps track of the name we are adding or updating
+const nameLocaleInputs: LocalizedNameInput = reactive(
+    { firstName: '',
+        lastName: '',
+        middleName: '',
+        locale: Locale.Und }
+)
```

Comment:
I think we could remove some of the comments by making sure the variable names are clear about what they represent. For example, change `addingLocaleName` to something along the lines of `shouldDisplayAddLocaleNameBtn`.